### DISCUSSION TITLE
Delete account

### DISCUSSION THREAD

**Discussion by dungng1321** [community/community#42067] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

Can i request to have my github account deleted due to policy violation . And Can I use the old account's gmail account to link to the new github account? 
Thank you !

**Comment by jfullstackdev** [comment]
try Support 

https://support.github.com/contact/account

as for your `email` maybe what you can do is to free it by adding another email that you will not be using then you are sure you can use the other email

**Reply by dungng1321** [reply, marked as answer]
Thank bro...but my github account has been suspended.. i can't login  my old account

**Comment by crralesandrini79** [comment]
Refund and close my account

**Comment by ghost** [comment]
Close my account
